docs/composables/useI18n.md

```diff
@@ -12,100 +12,100 @@ The `useI18n` composable returns an object containing several key methods and pr
 
 ### `$getLocale`
 
-- **Type**: `() => string`
-- **Description**: Returns the current locale of the application.
-- **Example**:
-  ```js
-  const { $getLocale } = useI18n()
-  const locale = $getLocale()
-  console.log(locale) // e.g., 'en'
-  ```
+-   **Type**: `() => Locale`
```

Comment:
It's better to keep `string` here to avoid breaking the existing logic, especially since you've already implemented `getLocaleCode`. Also, `getLocaleCode` should be added to the documentation.

Reply:
I'd suggest announce this breaking change before you are ready to merge it.

Changes made to `$getLocale` and `$getLocaleCode` have been added to the documentation.
And I moved some comments that shows the `Output` in the demostration code to a new section **Output**

Reply:
e.g. To make the documentation more clear and readable

Moved
**Example**

```typescript
console.log(1)
// Output: 1
```

To
**Example**

```typescript
console.log(1)
```

**Output**

```typescript
1
```

Reply:
The challenge isn’t just updating the code — the tests also need to be adjusted based on your changes. After reviewing, it seems simpler to keep the `getLocale` method returning a string, as the change doesn’t provide significant benefits. You can check the errors here: [GitHub Actions Errors](https://github.com/s00d/nuxt-i18n-micro/actions/runs/11343500624/job/31569601540?pr=34).

Reply:
understood

Reply:
Can you make corrections to the tests or return getLocale: string?
And need to resolve the conflicts; I made a couple of changes.

Reply:
May I close this pr? It seems I've made to much irrelevant changes in a single pr which makes it tricky
Would be happy to seperate them to several different prs

Reply:
> Can you make corrections to the tests or return getLocale: string? And need to resolve the conflicts; I made a couple of changes.
Here's the incoming prs
I'd like to implement a new function that returns a value with type Locale, keeping getLoacle: string
Add a new property displayName, and update relevant part of documentations
Improve documentations for making it more readable

Reply:
> Can you make corrections to the tests or return getLocale: string? And need to resolve the conflicts; I made a couple of changes.

Here's the incoming prs
I'd like to implement a new function that returns a value with type Locale, keeping getLoacle: string
Add a new property displayName, and update relevant part of documentations
Improve documentations for making it more readable